### ISSUE TITLE
Unable to Login to Excel2Json.IO

### ISSUE THREAD

**Issue by TTLGSYA** [brianlagunas/Excel2Json#38]
I got an "Internal Error" when just Sign-In to Excel2Json.IO; my email is "thanh-tung_le@kindermorgan.com", is it an invalid email from your perspective?  I need to have an Account to get the Save File option, thanks.

**Comment by brianlagunas** [comment]
Are you creating a new account, or are you trying to login into an existing account?

**Comment by TTLGSYA** [comment]
Thank you Brian for your help !

First, I tried to create a new Account, but I were not able to receive the Email to confirm; I have resent the Email Confirmation multiple times.
Secondly, I tried to Login and it seems that an Account is already created !  It’s recognized my ID and PW, but it as far as it goes.

In any case, please help me create and login to an Account.  According to some advices, if I can Sign-In to an Account, I have the Save File function, which I needed this function to see the URL of a Json File that are not residing on the Cloud.
Thanks.

Regards,
Thanh-Tung T. Le
Kinder Morgan Inc.
IT  Emerging Technologies
IT  Application Architect
***@***.******@***.***>
(713) 420 - 6022

**Comment by brianlagunas** [comment]
I would try just resetting your password. The account shows that it has been created and the email comfirmed

**Comment by brianlagunas** [comment]
Something else you can try and do is to clear your cookies and cache.

**Comment by brianlagunas** [comment]
Oh, I would make sure you have 3rd party cookies enabled for that site too.

**Comment by brianlagunas** [comment]
If you like, I can delete your account and you can try again.

**Comment by TTLGSYA** [comment]
Please delete and let me try again, thanks

Regards,
Thanh-Tung T. Le
Kinder Morgan Inc.
IT  Emerging Technologies
IT  Application Architect
***@***.******@***.***>
(713) 420 - 6022

**Comment by brianlagunas** [comment]
Account deleted. Let me know if you still have any issues. The Google Login should not give you any trouble :)
